openquake/calculators/tests/event_based_risk_test.py

```diff
@@ -433,8 +431,8 @@ def test_asset_loss_table(self):
         self.assertEqualFiles('expected/agg_curves_eb.csv', fname, delta=1E-5)
 
         curves = self.calc.datastore.read_df('agg_curves-rlzs',
-                                             ['NAME_1', 'return_periods'])
-        self.assertEqual(len(curves), 6)  # 2 names x 3 periods
+                                             ['site_id', 'return_periods'])
+        self.assertEqual(len(curves), 21)  # 7 sites x 3 periods
```

Comment:
It would be nice to keep both tests (for name_1 and site_id)

Reply:
Also, the exposure for test case_6c has different lon,lat coordinates for each asset, so we are not really testing the aggregation by site in this case

Reply:
I have changed the exposure to really test aggregation by site